platform/ui/src/components/FilterDateRange/FilterDateRange.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { DateRange, FilterWrapper } from '@ohif/ui';
+
+const FilterDateRange = ({
+  label,
+  isSortable,
+  isBeingSorted,
+  sortDirection,
+  onLabelClick,
+  inputValue,
+  inputProps,
+  onChange,
+}) => {
+  const { startDate, endDate } = inputValue;
+  return (
+    <FilterWrapper
+      label={label}
+      isSortable={isSortable}
+      isBeingSorted={isBeingSorted}
+      sortDirection={sortDirection}
+      onLabelClick={onLabelClick}
+    >
+      <div className="relative">
+        <DateRange
+          {...inputProps}
+          startDate={startDate}
+          endDate={endDate}
+          onChange={({ startDate, endDate }) => {
+            if (onChange) {
+              onChange({ startDate, endDate });
+            }
+          }}
+        />
+      </div>
+    </FilterWrapper>
+  );
+};
+
+FilterDateRange.defaultProps = {
+  label: '',
+  isSortable: false,
+  isBeingSorted: false,
+  sortDirection: 0,
+  onLabelClick: () => {},
+  inputValue: {},
+  inputProps: {},
+  onChange: () => {},
+};
+
+FilterDateRange.propTypes = {
+  label: PropTypes.string,
+  isSortable: PropTypes.bool,
+  isBeingSorted: PropTypes.bool,
+  sortDirection: PropTypes.number,
+  onLabelClick: PropTypes.func,
+  value: PropTypes.shape({
+    startDate: PropTypes.string,
+    endDate: PropTypes.string,
+  }),
+  inputProps: PropTypes.object,
```

Comment:
This is not specific enough. I know we're going to make these more explicit, but almost always want `shape` if we're going to support an object.

Reply:
I've removed it from the input components. Passing only the needed props